app/src/ui/lib/tooltip.tsx

```diff
@@ -300,6 +300,8 @@ export class Tooltip<T extends TooltipTarget> extends React.Component<
   }
 
   private onTargetMouseEnter = (event: MouseEvent) => {
+    this.mouseRect = new DOMRect(event.clientX - 10, event.clientY - 10, 20, 20)
```

Comment:
Maybe this can be extracted in to a function called from both `onTargetMouseEnter` and `onTargetMouseMove`?

Reply:
Refactor to rememberMousePosition.  Feel free to suggest a better name

Reply:
I'd just call it `updateMouseRect`, thanks!

Reply:
Updated